backend/compact-connect/lambdas/provider-data-v1/tests/function/__init__.py

```diff
@@ -137,7 +137,7 @@ def provider_jurisdictions_to_set(obj: dict):
                 Item=record
             )
 
-    def _generate_providers(self, *, home: str, privilege: str, start_serial: int):
+    def _generate_providers(self, *, home: str, privilege: str, start_serial: int, names: tuple[tuple[str, str]] = ()):
```

Comment:
nit: wouldn't a list of tuples be a more intuitive choice? I generally think of a tuple as fixed-size.

Reply:
Because this argument is optional and comes with a default value, it is a bad idea to [use a mutable type](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments) like a `List`.